tests/lit-tests/ssp.ispc

```diff
@@ -0,0 +1,16 @@
+// Make sure the appropriate ssp function attribute is set.
+
+// RUN: %{ispc} %s --emit-llvm-text -o - | FileCheck %s -check-prefixes=CHECK_ALL,DEFAULT
```

Comment:
It could make sense also verify that with different opt levels, including `-O0`.